lib/serve.js

```diff
@@ -294,6 +310,10 @@ module.exports = bootstrap(function createServer(options, config, configFile) {
         : 'No config is used'
     );
 
+    if (!options.prebuild) {
+        configFile.download = false; // disable static report downloading if not in prebuild mode
```

Comment:
I suppose that changing `configFile` will have no effect on model's config, since it already inherited on config normalize.

Reply:
This is the case when we starting cli without prebuild. So we do need this or the "Download report" button will be shown.

Reply:
Fixed